ffmpeg.yaml

```diff
@@ -0,0 +1,244 @@
+  - --enable-decoder=pcm_u8
+  - --enable-decoder=png
+  - --enable-decoder=rawvideo
+  - --enable-decoder=theora
+  - --enable-decoder=truehd
+  - --enable-decoder=vorbis
+  - --enable-decoder=vp8
+  - --enable-decoder=vp9
+  - --enable-decoder=wavpack
+  - --enable-decoder=webp
+  - --enable-decoder=wmalossless
+  - --enable-decoder=wmapro
+  - --enable-decoder=wmav1
+  - --enable-decoder=wmav2
+  - --enable-decoder=wmavoice
+  - --enable-demuxer=aac
+  - --enable-demuxer=ac3
+  - --enable-demuxer=asf
+  - --enable-demuxer=avi
+  - --enable-demuxer=bink
+  - --enable-demuxer=flac
+  - --enable-demuxer=gif
+  - --enable-demuxer=h264
+  - --enable-demuxer=image2
+  - --enable-demuxer=m4a
+  - --enable-demuxer=m4v
+  - --enable-demuxer=matroska
+  - --enable-demuxer=mjpeg
+  - --enable-demuxer=mov
+  - --enable-demuxer=mp3
+  - --enable-demuxer=ogg
+  - --enable-demuxer=pcm_s16le
+  - --enable-demuxer=wav
+  - --enable-encoder=aac
+  - --enable-encoder=ac3
+  - --enable-encoder=apng
+  - --enable-encoder=ffvhuff
+  - --enable-encoder=gif
+  - --enable-encoder=libopus
+  - --enable-encoder=libtheora
+  - --enable-encoder=libvorbis
+  - --enable-encoder=libvpx_vp8
+  - --enable-encoder=libvpx_vp9
+  - --enable-encoder=libx264
+  - --enable-encoder=mpeg4
+  - --enable-encoder=png
+  - --enable-encoder=wmav2
+  - --enable-filter=crop
+  - --enable-filter=fps
+  - --enable-filter=palettegen
+  - --enable-filter=paletteuse
+  - --enable-filter=scale
+  - --enable-hwaccel=h264_vaapi
+  - --enable-hwaccel=h264_vdpau
+  - --enable-hwaccel=hevc_vaapi
+  - --enable-hwaccel=hevc_vdpau
+  - --enable-hwaccel=mpeg4_vaapi
+  - --enable-hwaccel=mpeg4_vdpau
+  - --enable-hwaccel=vp9_vaapi
+  - --enable-indev=v4l2
+  - --enable-indev=xcbgrab
+  - --enable-muxer=ac3
+  - --enable-muxer=apng
+  - --enable-muxer=asf
+  - --enable-muxer=avi
+  - --enable-muxer=gif
+  - --enable-muxer=h264
+  - --enable-muxer=image2
+  - --enable-muxer=ipod
+  - --enable-muxer=mov
+  - --enable-muxer=mp4
+  - --enable-muxer=ogg
+  - --enable-muxer=opus
+  - --enable-muxer=webm
+  - --enable-parser=aac
+  - --enable-parser=aac_latm
+  - --enable-parser=ac3
+  - --enable-parser=dca
+  - --enable-parser=flac
+  - --enable-parser=h264
+  - --enable-parser=hevc
+  - --enable-parser=mjpeg
+  - --enable-parser=mpeg4video
+  - --enable-parser=mpegaudio
+  - --enable-parser=opus
+  - --enable-parser=vorbis
+  - --enable-parser=vp3
+  - --enable-parser=vp8
+  - --enable-protocol=file
+build-options:
+  arch:
+    i386:
+      config-opts:
+        - --enable-nvdec
+        - --enable-nvenc
+        - --enable-cuvid
+    x86_64:
+      config-opts:
+        - --enable-nvdec
+        - --enable-nvenc
+        - --enable-cuvid
+sources:
+  - type: archive
+    url: https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz
+    sha256: a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994
+post-install:
+  - install -p -D -m 0644 COPYING* "LICENSE.md" -t "${FLATPAK_DEST}/share/licenses/ffmpeg/";
```

Comment:
Same here for license.

Reply:
> License are not exported, shouldn't be installed.

Encouraging breaking the license should not take place here.

Reply:
> Same here for license.

As I said earlier, I'm not going to break the license.